src/Features/ERDDAP/Superlatives/index.tsx

```diff
@@ -29,17 +30,17 @@ export const Superlatives: React.FunctionComponent = () => {
 
   const lastHour = anHourAgoRounded()
```

Comment:
If we're not passing `lastHour` into `<ShowSuperlatives>`, then this can probably get removed.

With a backoff, we probably still want to pass `lastHour` in, and then tweak it to find the backoff time, as that should make it easier to have more predictable tests.

Reply:
Would the tests rely specifically on that `anHourAgoRounded()` function? I created the one that calculates the backoff depending on the number we pass it--would that be not super dependable testing wise?

Reply:
It's less about the function itself, it's more about where it's used. 

By having `lastHour` (or really the `Date.now()` within it) outside of the component that finds and renders the superlatives, we can set a different `lastHour` when testing, to make sure we match the times of the test data. By having `Date.now()` now being called inside the display component, we now would need to mock it to get the tests to be repeatable.

Reply:
Got it! Thanks for clarifying.